package.json

```diff
@@ -104,6 +104,7 @@
   "scripts": {
     "setup": "node ./setup/setup.js",
     "test": "node --max-old-space-size=4096 --trace-deprecation node_modules/jest-cli/bin/jest",
+    "test:update-snapshots": "yarn test -u",
```

Comment:
Could you change this to `yarn jest -u`? `yarn test` also runs the linting and this is not needed here.